CMakeLists.txt

```diff
@@ -34,6 +34,32 @@ endif()
 
 find_ros1_package(std_msgs REQUIRED)
 
+# find all known ROS1 message/service packages
+find_ros1_package(rosmsg REQUIRED)
+execute_process(
+    COMMAND bash -c "rosmsg list"
+    OUTPUT_VARIABLE rosmsg_output
+)
+execute_process(
+    COMMAND bash -c "rossrv list"
+    OUTPUT_VARIABLE rossrv_output
+)
+set(ros1_interfaces_string "${rosmsg_output}\n${rossrv_output}")
+
+string(REGEX REPLACE "\n" ";" ros1_interfaces_list "${ros1_interfaces_string}")
```

Comment:
Since the pattern doesn't use a regex this should be calling `string(REPLACE ...)`.